### PR TITLE
Reflect override settings for postfix

### PR DESCRIPTION
**Pull request by ineiti**
## What type of PR?

(documentation)

## What does this PR do?

Fixes path in the documentation setting, and also adds how to change postfix.cf

### CODE REVIEW

**Review by decentral1se — APPROVED** [review]
LGTM :rocket:

File: docs/faq.rst [hunk 1]
```diff
@@ -258,9 +258,11 @@ Postfix, Dovecot, Nginx and Rspamd support overriding configuration files. Overr
 correct syntax. The following file names will be taken as override configuration:
 
 - `Postfix`_ :
-   - ``main.cf`` as ``$ROOT/overrides/postfix/postfix.cf``
-   - ``master.cf`` as ``$ROOT/overrides/postfix/postfix.master``
+   - ``main.cf`` as ``$ROOT/overrides/postfix.cf``
```
Comment (Diman0) [inline docs/faq.rst:261]: The actual behaviour is that the overrides reside in $ROOT/overrides/postfix/postfix.cf.
This was changed in mailu 1.8.

File: docs/faq.rst [hunk 2]
```diff
@@ -258,9 +258,11 @@ Postfix, Dovecot, Nginx and Rspamd support overriding configuration files. Overr
...
+   - ``master.cf`` as ``$ROOT/overrides/postfix.master``
```
Comment (Diman0) [inline docs/faq.rst:262]: The actual behaviour is that the overrides reside in $ROOT/overrides/postfix/postfix.master.
This was changed in mailu 1.8.

**Review by Diman0 — CHANGES_REQUESTED** [review]
See the in line comments. The additional tip will be very handy for future users.

**Review by Diman0 — APPROVED** [review]
Looks good to me. Thank you for your contribution.

**Comment by ineiti** [comment]
OK, got confused in https://github.com/Mailu/Mailu/pull/1824 because I thought that 1.8 is the new stable. Now that I understand that 1.7 is stable, would it make sense to make a PR against 1.7 with the corrected paths for `postfix.cf` and `master.cf`?

**Review by lub — APPROVED** [review]
lgtm, thank you :)

PRs against previous branches are only opened after changes are merged in master. I don't think we have to backport it to 1.7 in particular, because hopefully 1.8 will be properly released soon.

**Comment by Diman0** [comment]
bors retry

**Comment by Diman0** [comment]
bors retry

**Pull request merged**